homeassistant/components/overkiz/sensor.py

```diff
@@ -454,7 +454,7 @@ def __init__(
     ) -> None:
         """Initialize the device."""
         super().__init__(device_url, coordinator)
-        self._attr_name = "HomeKit Setup Code"
+        self._attr_name = "HomeKit setup code"
```

Comment:
this could be moved below `_attr_entity_description`. Also, seems weird for this to be a sensor ?

Reply:
What do you suggest to display this HomeKit Setup Code instead of a sensor? This value is useful to help people to integrate their Somfy hub with the Homekit Controller. Somfy does not give this value for the old product.

Reply:
Somfy integration would discover a Homekit Controller config flow with the information pre-filled.

Reply:
Sorry, perhaps I missed this information. But how the Homekit Controller can automatically discovered the PIN code of the Somfy hub? If we added this sensor it was to help our end user to associate the hub to the Home Assitant Home Kit Controller.

Reply:
It doesn't yet, but that is the flow that we should prefer I think?

Reply:
For sure, if HA can read a property from an entity to automatically fill the HomeKit Controller association, it will be perfect.